examples/proxy-server/README.md

```diff
@@ -0,0 +1,22 @@
+# Armeria proxy server
+
+A proxy server that balances individual requests across the healthy backends using 
+[Client-side load balancing](https://line.github.io/armeria/client-service-discovery.html).
+
+## Demo
+
+To startup the server, use Gradle: 
+
+```bash
+$ ./gradlew run --no-daemon
+```
+
+This will start up the proxy server and three backend servers which send streaming responses.
+Each backend server generates streaming data of ASCII art at different interval.
```

Comment:
Each backend server streams ASCII animation to the proxy server at different interval.